src/views/UpdateLocusGeneDiseaseView.vue

```diff
@@ -128,9 +133,13 @@ export default {
         .then((responseJson) => {
           this.isPreviousInputDataLoading = false;
           this.previousInput = prepareInputForUpdating(responseJson.data);
+          this.publications = this.previousInput.publications || [];
```

Comment:
dont think this line is required, since prepareInputForUpdating already sets it to {} if there no publications for the saved draft